src/components/NFTproduct/Nftproduct.tsx

```diff
@@ -199,32 +203,38 @@ const Nftproduct =  () => {
           </BuyNow>
         </BuyCard>
 
-        <PublisherCard height='91px'>
+        <PublisherCard height="91px">
           <OwnerText>
-            <Text fontFamily='Roboto' FontWeight='400' fontSize='1rem' color='#323232' margin='15px 0px 0px 22px'>
-
+            <Text fontFamily="Roboto" FontWeight="400" fontSize="1rem" color="#323232" margin="15px 0px 0px 22px">
               {t('Originally Crated By')} :
-
+            </Text>
+            {auctionNFT ? (
+              <Text fontFamily="Roboto" FontWeight="600" fontSize="1rem" color="#323232" margin="15px 0px 0px 0px">
+                {StringModifier(auctionNFT?.nft?.issuer)}
               </Text>
-            {auctionNFT ? <Text fontFamily='Roboto' FontWeight='600' fontSize='1rem' color='#323232' margin='15px 0px 0px 0px'>{StringModifier(auctionNFT?.nft?.issuer)}</Text> : null}
+            ) : null}
           </OwnerText>
           <OwnerText>
-            <Text fontFamily='Roboto' FontWeight='800' fontSize='1rem' color='#323232' margin='30px 0px 0px 22px '>
+            <Text fontFamily="Roboto" FontWeight="800" fontSize="1rem" color="#323232" margin="30px 0px 0px 22px ">
```

Comment:
if it Issue Royalty Share is zero should be replace it with message that "no royalty share" or another common message

Reply:
fixed